src/connection.ts

```diff
@@ -38,7 +38,7 @@ const isErrorEvent = (res: WebSocket.CloseEvent | WebSocket.Data | WebSocket.Err
  */
 export class StableWSConnection<StreamChatGenerics extends ExtendableGenerics = DefaultGenerics> {
   // global from constructor
-  client: StreamChat<StreamChatGenerics>;
+  client?: StreamChat<StreamChatGenerics>;
```

Comment:
Changes in this file are related to https://github.com/GetStream/stream-chat-js/pull/1029/files#r943398458